test/test_apidb_backend_changesets.cpp

```diff
@@ -857,10 +859,26 @@ void test_changeset_close(test_database &tdb) {
 
 } // anonymous namespace
 
-int main(int, char **) {
+int main(int argc, char** argv) {
```

Comment:
Somehow this introduces a bit of code duplication across test cases. I do realize that this would be useful for cmake. Since it's not needed right now for the current autotools based build, I'd like to keep this commit separate from the others. Either I could cherry-pick all other commits and omit this one, or you could split this up on your end. Whatever works best for you.

Reply:
Yeah, it could probably be refactored. I removed it from this PR for now.